### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: parquet/fuzz/fuzz_targets/parse_metadata.rs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#![no_main]
```
Comment (xansec) [inline parquet/fuzz/fuzz_targets/parse_metadata.rs:1]: This target seems to be for parquet, and not arrow-rs.

**Review by xansec — CHANGES_REQUESTED** [review]
Is this part of arrow-rs, or parquet? If the latter it should be a target for parquet. Please make the requested changes.

File: parquet/fuzz/fuzz_targets/parse_metadata.rs (hunk 1 above)
Comment (evanrichter) [reply parquet/fuzz/fuzz_targets/parse_metadata.rs:1]: > Is this part of arrow-rs, or parquet?

this implementation of parsing the parquet format is part of this arrow-rs project. As I understand it, parquet is the file format, and arrow is the in-memory format.

You might be confused with [arrow2](https://github.com/jorgecarleitao/arrow2#faq)? which is a significantly divergent fork with a very different approach to the format.

Comment (xansec) [inline parquet/fuzz/fuzz_targets/parse_metadata.rs:1]: > This crate contains the official Native Rust implementation of Apache Parquet, which is part of the Apache Arrow project. \n Ok, this makes sense to me.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**